core/src/test/java/hudson/PluginWrapperTest.java

```diff
@@ -10,36 +10,35 @@
 import java.util.jar.Manifest;
 
 import jenkins.model.Jenkins;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.*;
 import org.mockito.invocation.InvocationOnMock;
 import org.mockito.stubbing.Answer;
 
+import org.jvnet.hudson.test.Issue;
```

Comment:
I am not sure if this is used

Reply:
It is, because of Line 198:    @Issue("JENKINS-52665")

Reply:
Ahhh, nevermind then :)